config/locales/en/course/assessments.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  activerecord:
+    errors:
+      models:
+        course/assessment:
+          no_questions: 'Assessment can exit draft mode only after adding questions'
```

Comment:
I THINK that this should be a sentence fragment. Can you trigger the validation failure and see how simple form presents the error message?